config.json

```diff
@@ -2357,6 +2357,23 @@
           "macros"
         ],
         "difficulty": 8
+      },
+      {
+        "slug": "pov",
+        "name": "POV",
+        "uuid": "??? Do I need to create one? How about: 7",
```

Comment:
Yes, absolutely, every exercise requires a uuid-v4 which is unique across the whole site. `uuidgen` in your terminal or googling uuid-v4 generator will help you to get one.

Reply:
Thank you!